packages/link/src/__demo__/examples/link.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright 2017 CA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+import { mineralTheme } from '@mineral-ui/component-utils';
+import Link from '../../Link';
+
+export default {
+  title: 'Basic link',
+  description: 'an anchor tag',
+  scope: { Link },
+  source: `<Link to="https://example.com">Make it so</Link>`
+};
```

Comment:
This example also does not work as expected.  It goes to `http://0.0.0.0:8080/https://example.com`